.gitignore

```diff
@@ -7,12 +7,10 @@ cmake-build-*/
 .vs/
 .idea/
 .vscode/
-!.vscode/settings.json
```

Comment:
Please revert this change. Many people commit their workspace and project settings so others on the team have access to the same settings and debug commands